tests/conftest.py

```diff
@@ -417,6 +421,17 @@ def generate_greedy_logprobs_limit(
     def encode(self, prompts: List[str]) -> List[List[torch.Tensor]]:
         return self.model.encode(prompts)
 
+    def encode_simple(
```

Comment:
Imo the naming is a bit misleading since you're returning the logits of the classification head, not the encoded input directly.

Reply:
Yes, you are right, I use encode_* because I did't find a proper func name for this general 'any' model scenario, maybe someone could give me some suggestion.

Reply:
Hmm I think since this is so general, you can extract the logits inside your specific test rather than in `conftest.py`. I'm tempted to just call it `forward` since all it does is pass the inputs directly to the model.

Reply:
Good idea, now it's late, I'll refactor it tomorrow, thank for your advice :)